coinflip/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel = "stylesheet" href = "styles.css">
+</head>
+  
+<body>
+    <div class = "container">
+   
+<!-------- Coin div ----------------->
+    <div class = "coin" id = "coin">
+        <div class = "heads">
+    <img src = "images/silverheadscoin.png">
+    </div>
+     <div class = "tails">
+        <img src = "images/goldtailscoin.png">
+    </div>
+   </div>
+   <div class = "stats">
+    <p id = "heads-count"> Heads: 0</p>
+    <p id = "tails-count"> Tails: 0</p>
+</div>
+<div class = "buttons"></div>
+<button id = "flip-button">Flip Coin</button>
```

Comment:
buttons should always have a type, in this case it would be `type=button` buttons default to `type=submit`, and that should only be used for forms